iwf-sdk.yaml

```diff
@@ -497,6 +497,7 @@ components:
         - LOAD_ALL_WITHOUT_LOCKING
         - LOAD_PARTIAL_WITHOUT_LOCKING
         - LOAD_PARTIAL_WITH_EXCLUSIVE_LOCK
+        - LOAD_EMPTY
```

Comment:
I feel like it should be `None` rather than `Empty`. Empty means the result is empty. What we want to tell is to not load anything

Reply:
Makes sense!